docs/synapse/user_interface.rst

```diff
@@ -0,0 +1,34 @@
+.. _synapse-ui:
+
+Synapse User Interface
+======================
+
+
+The Synapse UI is a WebUI which integrates with a Cortex to provide a view into your Synapse
+Ecosystem. A few important UI features include:
+
+    - Directly supports collaborative analysis. Users can view changes made by their teammates immediately. This enables
+      them to rapidly work together as peers.
+
+    - Simplified Storm query bar and point-and-click navigation for exploring data. User can visualize their data in
```

Comment:
Sounds like this should be "A user" or "Users"

Reply:
fixed